.github/workflows/python-cross-platform-support.yml

```diff
@@ -91,59 +91,20 @@ jobs:
       - name: Get Package Version
         run: python -m aac version
 
-      - name: Validate Core Spec
+      - name: Check AaC DSL
         if: matrix.os != 'windows-latest'
         run: |
-          spec_file=$(find $pythonLocation -name spec.yaml | head -n1)
-          python -m aac validate $spec_file
+          spec_file=$(find $pythonLocation -name aac.aac | head -n1)
+          python -m aac check $spec_file
 
-      - name: Validate Core Spec
+      - name: Check AaC DSL
         if: matrix.os == 'windows-latest'
         run: |
-          $spec_file=$(gci -r -fi spec.yaml -Name "$Env:pythonLocation")
-          python -m aac validate "$Env:pythonLocation\$spec_file"
+          $spec_file=$(gci -r -fi aac.aac -Name "$Env:pythonLocation")
+          python -m aac check "$Env:pythonLocation\$spec_file"
 
       - name: Print out Core Spec
-        run: python -m aac print-spec
-
-      - name: Print out active context
-        run: python -m aac print-active-context
-
-      - name: Validate Primitive Type Check Plugin
-        run: python -m aac validate src/aac/plugins/first_party/primitive_type_check/primitive_type_check.yaml
-
-      - name: Validate Example Model
-        run: python -m aac validate model/flow/System.yaml
-
-      - name: Validate Alarm Clock
-        run: python -m aac validate model/alarm_clock/alarm_clock.yaml
-
-      - name: Create PlantUML Component Diagram
-        run: python -m aac puml-object model/flow/System.yaml output/
-
-      - name: Test Gen-Plugin Third-Party Plugin
```

Comment:
Not sure I understand...help me out.

Reply:
I think this might actually fall under similar testing shift as above about checking models and I've just not hit the new testing suite in my review yet to know for sure. 

But how we had steps exercising the `gen-plugin` functionality to make a first and third party plugin, would we not want a step exercising the new generate functionality?